setup.py

```diff
@@ -153,15 +159,15 @@ def read_file(*path):
       tests_require = tests_require,
       extras_require = dict(test=tests_require),
       install_requires = [
-        'persistent',
-        'BTrees',
+        'persistent >= 4.1.0',
+        'BTrees >= 4.1.3',
         'ZConfig',
-        'transaction >= 1.4.1' if PY3 else 'transaction',
+        'transaction >= 1.4.4',
         'six',
         'zc.lockfile',
         'zdaemon >= 4.0.0a1',
         'zope.interface',
-        ] + (['zodbpickle >= 0.2'] if PY3 else []),
+        ] + (['zodbpickle >= 0.6.0'] if (PY3 or PY27 or PYPY) else []),
       zip_safe = False,
```

Comment:
Good question. As far as I can see, no, it shouldn't. The default protocol didn't change, and zodbpickle is based on the pickle modules from Python 2.7 anyway. The fancy string encoding stuff zodbpickle can do is only used under Python 3.